fastapi_profiler/profiler.py

```diff
@@ -44,6 +49,15 @@ def __init__(
                 "to set shutdown event handler to output profile."
             )
 
+        if profiler_output_type == "prof":
+            self._profiler = cProfile.Profile()
+            self._start_profiler = self._profiler.enable
+            self._stop_profiler = self._profiler.disable
+        else:
+            self._profiler = Profiler(interval=profiler_interval, async_mode=async_mode)
+            self._start_profiler = self._profiler.start
+            self._stop_profiler = self._profiler.stop
```

Comment:
Since I'm loading the `cProfile` into the same variable the `pyinstrument` Profiler would otherwise occupy, I set up function aliases here to make sure I call the right once since the interface is slightly different

Reply:
LGTM~